### PR TITLE
Fixing scancode issues.

### PR DESCRIPTION
**Pull request by jthomas**
See https://travis-ci.org/apache/incubator-openwhisk-pluggable-provider/builds/535290433?utm_source=github_status&utm_medium=notification for issues.

Added licenses to all outstanding files identified and removed trailing whitespace where found.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# A Travis CI configuration file.
```
Comment (rabbah) [inline .travis.yml:1]: might as well just use the long form.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
```
Comment (rabbah) [inline gradle/wrapper/gradle-wrapper.properties]: here also.

**Review by rabbah — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (jthomas) [reply .travis.yml:1]: scancode.py does not accept the full license header in these files yet?

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (mrutkows) [reply gradle/wrapper/gradle-wrapper.properties]: Scancode does not enforce mini or full against any specific file type... it just cares one of the other is present in the top X lines of the source file.

Comment (jthomas) [reply gradle/wrapper/gradle-wrapper.properties]: My bad - this was a user error. I've now fixed this as instructed.

File: .travis.yml (hunk 1 above)
Comment (jthomas) [reply .travis.yml:1]: My bad - this was a user error. I've now fixed this as instructed.

**Pull request merged**